resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -107,7 +107,6 @@ object DbReader {
   // Note that we explicitly check nodepools with 5/11 of statuses that exist.
   // The statuses we exclude are PROVISIONING, STOPPING, DELETED, PRECREATING, PREDELETING, and PREDELETING
   //    - We exclude all -ING statuses because they are transitionary, and the purpose of this is not to handle timeouts
-  //    - Unclaimed we exclude because we never want to clean up batch created nodepools
```

Comment:
Wasn't there something in `applessNodepoolQuery` that disallowed batch-created nodepools to be selected by the query?

Reply:
I see a test for this a bit unecessary since the status in question (NodepoolStatus.Unclaimed) was removed from leo entirely

Reply:
Yeah, a change was needed for applessNodepoolQuery https://github.com/broadinstitute/leonardo-cron-jobs/pull/63/files#diff-7bd14f4fe178f136127d110beb4c945537788ea7bc7f4c066245c660219a9427R163

Reply:
The change you linked to is for `kubernetesClustersToDeleteQuery`, not `applessNodepoolQuery`. 

Do you remember why the comment

> Unclaimed we exclude because we never want to clean up batch created nodepools

was put there at the first place? If it was there by mistake, we're good but if not, it'd make me wonder if the query needs updating to not do any exclusion anymore.

Reply:
Ah, sorry. Yeah, so the query explicitly selects the statuses we want to include. 'Unclaimed' is not there for the reason described, which is no longer relevant, so only a comment update was necessary. The query remains correct still.

Reply:
Ah now I understand - thank you.